projects/components/src/tabs/navigable/navigable-tab-group.component.ts

```diff
@@ -57,7 +68,8 @@ export class NavigableTabGroupComponent implements AfterContentInit {
   public ngAfterContentInit(): void {
     this.activeTab$ = merge(this.navigationService.navigation$, this.tabs.changes).pipe(
       startWith(undefined),
-      map(() => this.findActiveTab())
+      map(() => this.findActiveTab()),
+      tap(activeTab => this.tabChange.emit(activeTab?.path))
```

Comment:
I wonder if this would emit duplicate events. May be we can put `distinctUntilChanged` operator before the tap.